data/src/main/java/org/tiqr/data/util/extension/JavaByteArray.java

```diff
@@ -0,0 +1,112 @@
+package org.tiqr.data.util.extension;
+
+/**
+ * Do not convert this to Kotlin to keep the implementation exactly the same.
+ */
+public final class JavaByteArray {
```

Comment:
There are multiple versions of this on develop, which fix a bug on production, did you also integrate those?

Reply:
That's the next step. Should have added a comment about it.